_posts/2023-11-02-election-competitiveness-in-charlotte-north-carolina/election-competitiveness-in-charlotte-north-carolina.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Election Competitiveness in Charlotte, North Carolina"
+description: |
+  Analyzing changes in competitiveness for mayoral and city council elections from 2015 to 2022 in Charlotte, North Carolina.
+author: Angeline Zhao
+date: 2023-11-02
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## City Council in Charlotte
+
+The Charlotte City Council is made of the Mayor, four at-large city council members, and one council member for each of the seven city council districts. Currently, two members of the city council are Republicans and the remaining are Democrats.
+
+For the demographic breakdown, Charlotte is roughly 40% White, 33% Black, 16% Hispanic, and 7% Asian. For the partisan breakdown, Charlotte voted 68% Democrat and 32% Republican in the the latest 2021 Mayoral Election.
+
+## Decreased Competition in Charlotte Elections
+
+Charlotte, NC is a fascinating city where voting patterns have undergone significant changes in the last few years alone. For example, in the 2015 Mayoral election, the partisan breakdown was 52% Democrat and 48% Republican, which is much more competitive than the most recent election, which voted 68% Democrat and 32% Republican. The significant increase in Democrat voters and decrease in competitiveness raises the question of whether the recently enacted city council districts influenced the competitiveness in Charlotte or if the city has simply become more left-leaning over time.
+
+## Analyzing Competitiveness In 2015 and 2022 Elections On Simulated District Maps
+
+I used election data from the 2022 Charlotte Mayoral and city council at-large elections and the 2015 Mayoral election to generate two sets of simulated redistricting plans using the Sequential Monte Carlo (SMC) algorithm. For each simulated plan, a measure of competitiveness in each district was calculated by the absolute value of the difference in Democrat and Republican votes divided by the total number of votes. This measure of competitiveness allowed me to compare only whether districts were more competitive without focusing on which party it favored. In this case, the measure of competitiveness is always between zero and one, where lower values indicate greater competition.
+
+I calculated the competitiveness by district for simulated plans based on 2022 Mayoral election and city council at-large election data to see if the enacted plan was more or less competitive than the simulations. I also plotted the competitiveness by district based on the 2015 Mayoral election results, still with the currently enacted plan, to see if there are significant differences in competitiveness when these two sets of election results are simulated according to the same map.
+
+![](comp_22_abs.png){width="319"}
+
+![](comp_15_abs.png){width="320"}
```

Comment:
Could you say more about what you mean here? I think Chris had the same question: I thought you had one set of simulations you were comparing against two reference plans. Here, the boxplots look different: do you have two sets of different simulations?

Reply:
I know we clarified the 2 sets of simulations during the meeting, but I'm looking at the red bars and they look different between the 2 graphs. I thought the reference plans were supposed to be the same across the 2 graphs?